Plugins/Wox.Plugin.Program/Programs/UWP.cs

```diff
@@ -254,23 +254,23 @@ public class Application : IProgram
             public string LogoPath { get; set; }
             public UWP Package { get; set; }
 
-            private int Score(string query)
+            private int Score(string query, bool shouldUsePinYin)
```

Comment:
Would it be much cleaner if you just use: Main._settings.SkipPinyin ? StringMatcher.ScoreForPinyin : 0 
Above would save you from passing the settings object/ the shouldusePinYin parameter around all over. The 'setting' members on both UWP and Win32  is internal static, which makes sense and can be used with direct calling Main._settings